### ISSUE TITLE
Cannot declare different sized `bn::bitset` at the same time

### ISSUE THREAD

**Issue by copyrat90** [GValiente/butano#23]
```cpp
#include "bn_bitset.h"
#include "bn_core.h"

int main() {
    bn::core::init();
    bn::bitset<16> flags;
    bn::bitset<32> another_flags;
    while (true)
        bn::core::update();
}
```

```bash
In file included from ../src/main.cpp:1:
../butano/include/bn_bitset.h: In instantiation of 'class bn::bitset<32>':
../src/main.cpp:7:20:   required from here
../butano/include/bn_bitset.h:716:43: error: ambiguating new declaration of 'constexpr bn::bitset<32> bn::operator&(const ibitset&, const ibitset&)'
  716 |     [[nodiscard]] constexpr friend bitset operator&(const ibitset& a, const ibitset& b)
      |                                           ^~~~~~~~
../butano/include/bn_bitset.h:716:43: note: old declaration 'constexpr bn::bitset<16> bn::operator&(const ibitset&, const ibitset&)'
../butano/include/bn_bitset.h:726:43: error: ambiguating new declaration of 'constexpr bn::bitset<32> bn::operator|(const ibitset&, const ibitset&)'
  726 |     [[nodiscard]] constexpr friend bitset operator|(const ibitset& a, const ibitset& b)
      |                                           ^~~~~~~~
../butano/include/bn_bitset.h:726:43: note: old declaration 'constexpr bn::bitset<16> bn::operator|(const ibitset&, const ibitset&)'
../butano/include/bn_bitset.h:736:43: error: ambiguating new declaration of 'constexpr bn::bitset<32> bn::operator^(const ibitset&, const ibitset&)'
  736 |     [[nodiscard]] constexpr friend bitset operator^(const ibitset& a, const ibitset& b)
      |                                           ^~~~~~~~
../butano/include/bn_bitset.h:736:43: note: old declaration 'constexpr bn::bitset<16> bn::operator^(const ibitset&, const ibitset&)'
```

**Comment by GValiente** [comment]
It should be fixed in [10.6.1](https://github.com/GValiente/butano/releases/tag/10.6.1).

**Comment by copyrat90** [comment]
Thanks as always!
